### PR TITLE
Equailizer

### PR DESCRIPTION
**Pull request by joacomiguens**
### Summary

- Create equalizer.scss.

- Create styles.scss

- Pass styles on application.scss to styles.scss.

- Create colors.scss

### Screenshot

image

### Trello Card

You can find it [here](https://trello.com/c/B2rK7OLL/17-css-resets)

### CODE REVIEW

File: scss/_styles.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap");
```
Comment (ebedoyaalzate) [inline scss/_styles.scss]: You don`t need this import.

File: scss/_styles.scss [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+@import "colors";
```
Comment (ebedoyaalzate) [inline scss/_styles.scss]: Use single quote

File: scss/application.scss [hunk 3]
```diff
@@ -0,0 +1,2 @@
+@import "equalizer";
+@import "styles";
```
Comment (ebedoyaalzate) [inline scss/application.scss]: Same here

File: scss/_styles.scss (hunk 2 above)
Comment (joacomiguens) [reply scss/_styles.scss]: Done 😃

File: scss/application.scss (hunk 3 above)
Comment (joacomiguens) [reply scss/application.scss]: Done 😃

**Review by mcallegari10 — APPROVED** [review]

**Pull request merged**